pyccel/codegen/printing/fcode.py

```diff
@@ -2640,15 +2640,14 @@ def _print_Header(self, expr):
 
     def _print_SysExit(self, expr):
         code = ""
-        if expr.status.dtype is not NativeInteger() or expr.status.rank > 0:
+        if isinstance(expr.status, LiteralInteger):
+            arg = str(expr.status)
+        elif expr.status.dtype is not NativeInteger() or expr.status.rank > 0:
             print_arg = FunctionCallArgument(expr.status)
             code = self._print(PythonPrint((print_arg, ), file="stderr"))
             arg = "1"
         else:
-            arg = expr.status
-            if arg.precision != 4:
-                arg = NumpyInt32(arg)
-            arg = self._print(arg)
+            arg = str(expr.status)
```

Comment:
I'm not sure that this change is a good idea. The `if` above will handle the literal integer case, but for non-literal integers (e.g. expressions/variables) we should keep the conversion and the call to `_print`. Does this pass the tests as written?